src/community/ogcapi/ogcapi-core/src/main/java/org/geoserver/ogcapi/APIConformance.java

```diff
@@ -0,0 +1,113 @@
+package org.geoserver.ogcapi;
+
+import java.util.Objects;
+
+/**
+ * Service capability or feature, identified by conformance class.
+ *
+ * OGCAPI Web Services are defined with core functionality, strictly extended with additional, optional, functionality
+ * identified by "conformance class".
+ *
+ * OGC Open Web Services can be extended using application profiles with additional, optional, functionality.
+ */
+public class APIConformance {
+
+    /**
+     * Standards approval status.
+     */
+    public enum Status {
+        APPROVED, DRAFT, INFORMAL
+    }
+
+    public enum Type { CORE, EXTENSION }
+
+    final APIConformance parent;
```

Comment:
Do conformances have a "parent"? Don't think they extend in terms of inheritance, but surely some make sense only when a given set of other conformance classes have been implemented (eventually just one, typically the "Part 1, core" of the relevant service).

Some however exist on their own, typically the ones in OGC API commons. E.g., the JSON conformance class for OGC services is recommended but not required.

That said, wondering if it's something we need to communicate. I see the field, but no getter, and no usage.

Reply:
There is a factory method, you can see its use here:

```
    public static final APIConformance CORE = new APIConformance("http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/core");
    public static final APIConformance FEATURES_FILTER = CORE.extend(ConformanceClass.FEATURES_FILTER);
```

In reading the standards they often say they are an extension of a specific core.